src/home_robot_hw/home_robot_hw/remote/modules/nav.py

```diff
@@ -90,18 +96,53 @@ def wait_for_waypoint(
             # Loop until we get there (or time out)
             curr = self.get_base_pose()
             pos_err = np.linalg.norm(xy - curr[:2])
+            rot_err = angle_difference(curr[-1], xyt[2])
             if verbose:
-                print("- curr pose =", curr, "target =", xyt, "err =", pos_err)
-            if pos_err < pos_err_threshold:
+                print(
```

Comment:
If you haven't seen self-documenting f-strings they're pretty convenient! `print(f"- {curr pose=} {target =}...")`

Reply:
had not seen this, super useful